qiskit/quantum_info/operators/symplectic/pauli_list.py

```diff
@@ -401,7 +401,7 @@ def insert(self, ind, value, qubit=False):
         Args:
             ind (int): index to insert at.
             value (PauliList): values to insert.
-            qubit (bool): if True delete qubit columns, otherwise delete
+            qubit (bool): if True insert qubit columns, otherwise insert
                           Pauli rows (Default: False).
```

Comment:
```suggestion
            qubit (bool): if ``True`` insert qubit columns, otherwise insert
                          Pauli rows (Default: ``False``).
```